test/python/test_rsqrtgrad.py

```diff
@@ -0,0 +1,61 @@
+# ==============================================================================
+#  Copyright 2018-2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge ReluGrad operation test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+
+import tensorflow as tf
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops.gen_math_ops import rsqrt_grad
+
+from common import NgraphTest
+import numpy as np
+
+
+class TestRsqrtGrad(NgraphTest):
+
+    def test_rsqrtgrad_2d(self):
+        y = constant_op.constant(
+            self.generate_random_numbers(6, 1.0, 10.0), shape=[2, 3])
+        dy = constant_op.constant(
+            self.generate_random_numbers(6, 0.0, 100.0), shape=[2, 3])
+
+        out = rsqrt_grad(y, dy)
+
+        def run_test(sess):
+            return sess.run(out)
```

Comment:
Using constants might constant fold. Please run `NGRAPH_TF_LOG_PLACEMENT=1` to see if it constant-folds. We can use placeholders for `y` and `dy`

Reply:
I checked the placement already :) 
```OP_placement:	Host	_SOURCE (NoOp)
OP_placement:	Host	_SINK (NoOp)
OP_placement:	Host	_retval_RsqrtGrad_0_0 (_Retval)
OP_placement:	nGraph[0]	Const (Const)
OP_placement:	nGraph[0]	Const_1 (Const)
OP_placement:	nGraph[0]	RsqrtGrad (RsqrtGrad)```